nodemon.json

```diff
@@ -1,6 +1,6 @@
 {
   "watch": ["app"],
-  "ext": "ts, json",
+  "ext": "ts, json, ejs",
```

Comment:
Nitpick, but `ejs`-files are re-rendered on every request, so this is probably unnecessary.